source/services/dataContracts/converters/aliasConverter/aliasConveter.tests.ts

```diff
@@ -0,0 +1,20 @@
+import { AliasConverter } from './aliasConverter';
+
+describe('aliasConverter', (): void => {
+	let aliasConverter: AliasConverter<string>;
+
+	beforeEach((): void => {
+		aliasConverter = new AliasConverter<string>('propValue');
+	});
+
+	it('should get the property from the parent context using the specified property name', (): void => {
+		let parent: any = { propValue: 'value' };
+		expect(aliasConverter.fromServer(undefined, parent)).to.equal('value');
+	});
+
+	it('should set the value on the parent context using the specified property name', (): void => {
+		let parent: any = {};
+		aliasConverter.toServer('value', parent);
```

Comment:
On the bright side the object will already get cloned before the transforms get applied to it, as per https://github.com/RenovoSolutions/TypeScript-Angular-Utilities/pull/107/commits/95ec85a0b4a100758a676dac74c4b37a9571beb1#diff-3849d8b1880fcba661588f066c3c218bR35